README.md

```diff
@@ -1,2 +1,74 @@
-# oam-runtime
-A set of libraries for building OAM runtimes
+# oam-kubernetes-runtime Project
+
+The OAM Kubernetes runtime project is a collection of Golang help libraries and official api type
+ definitions for building an OAM Kubernetes runtime that is compatible with the latest stable
+  OAM spec. 
+
+## Prerequisites
+
+1. go version 1.12+
+2. Kubernetes version v1.15+ with KubeCtl configured 
+
+
+## Getting started
+
+The functionality of this library can be demonstrated with the following steps:
+
+* Fork and Clone this project
+
+* Build the library 
+
+```shell
+make submodules 
+
+make generate
+
+make
+```
+
+* Install CRDs to your Kubernetes cluster
+
+```shell
+kubectl apply -f cluster/charts/oam-kubernetes-runtime/crds/
+
+```
```

Comment:
These are being removed as it makes more sense for an implementation to generate their own CRDs

Reply:
in this case, I will add a make CRD command in the make file so that this example can be self-contained.